vendor/origoelec/ED10W-codec.yaml

```diff
@@ -0,0 +1,60 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: windsensor.js
```

Comment:
@origoelec have you the decoder file? if not please delete this file for the moment, it will throw an error if you leave it.